mappings/net/minecraft/util/function/CharPredicate.mapping

```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/class_5462 net/minecraft/util/function/CharPredicate
```

Comment:
Hmm, I recall it is more specific than that in the usage in `Util` class, though i may be wrong

Reply:
Should be in `client.util` imo

Reply:
Humm, ill check to see if it is client only, if so ill move it.

Reply:
Yes, moved.